figures/generate-figures.sh

```diff
@@ -59,13 +59,11 @@ bash ${analyses_dir}/chromothripsis/run-chromothripsis.sh
 OPENPBTA_CNS_FIT_ONLY=1 bash ${analyses_dir}/mutational-signatures/run_mutational_signatures.sh
 
 # Run the collapse-rnaseq module, which is needed for telomerase, immune deconvolution, and GSVA modules
-OPENPBTA_TP53_FIGURES=1 bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh 
+bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh 
```

Comment:
Should we only add the specific scripts for collapse counts (as above in the `RUN` script), given that we have collapsed FPKM already in the data folder and this, GSEA, and immune deconv should be utilizing data for FPKM collapsed, not local.

Reply:
also cc @jaclyn-taroni here who added `OPENPBTA_TP53_FIGURES=1` - I think this happened during the release, but I cannot recall exactly why

Reply:
I didn't add this here. @sjspielman did in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1454 ([blame](https://github.com/AlexsLemonade/OpenPBTA-analysis/blame/master/figures/generate-figures.sh)).

I did add it to `analyses/tp53_nf1_score/run_classifier.sh` (which makes more sense) here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1394

Reply:
I have absolutely no explanation for why I added that `OPENPBTA_TP53_FIGURES`!

Reply:
> Should we only add the specific scripts for collapse counts (as above in the RUN script), given that we have collapsed FPKM already in the data folder and this, GSEA, and immune deconv should be utilizing data for FPKM collapsed, not local.

I'm not exactly sure what you mean here, can you clarify? I'll also note this PR (haven't requested review yet since it's not meant to be merged for a while) in case it's relevant to this comment? https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1470

Reply:
Yes. I meant that I don't think we need to run the whole `collapse-rnaseq` module in this script since TP53, GSEA, immune, and part of this module which utilize the `pbta-gene-counts-rsem-expected_fpkm-collapsed*` are doing so from the release via `data/` and not the local module directory. Instead, since this module is the only one to _additionally_ utilize the `pbta-gene-counts-rsem-expected_counts-collapsed*` files, I think we can add the limited code below to the figure generation script:

```bash
Rscript ../collapse-rnaseq/01-summarize_matrices.R -i ../../data/pbta-gene-counts-rsem-expected_count.stranded.rds -g ../../data/gencode.v27.primary_assembly.annotation.gtf.gz -m ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.stranded.rds -t ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed_table.stranded.rds
Rscript ../collapse-rnaseq/01-summarize_matrices.R -i ../../data/pbta-gene-counts-rsem-expected_count.polya.rds -g ../../data/gencode.v27.primary_assembly.annotation.gtf.gz -m ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.polya.rds -t ../collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed_table.polya.rds
```

Thoughts?

Reply:
This idea is why I added `OPENPBTA_FOR_FIGURES=1` on line 65, as well as in the telomerase run script, so I think the change you are asking about has already been made in this PR. I added this option to be able to only run the line needed to generate stranded FPKM, which is the only result used in figures.

Reply:
Oh, gotcha, but then is line 62 not needed at all in that case?
`bash ${analyses_dir}/collapse-rnaseq/run-collapse-rnaseq.sh`

Reply:
Checking back to some of the discussion in #1455, no it is not needed. But, that sort of change of wholesale removing an analysis module from the script would be more appropriately done in #1470, not this PR. I will make that change now over in that branch.